Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM continuumio/miniconda3
```

Comment:
we may need to change this, but for now I think this is great

Reply:
OK! I agree hahaha